app/services/foreman_gutterball/gutterball_service.rb

```diff
@@ -26,24 +26,14 @@ class GutterballService < GutterballResource
     def initialize(_options = {})
```

Comment:
you can also just delete this `initialize` method if we're not doing anything with it.